src/components/JobsTrainingContent/index.jsx

```diff
@@ -0,0 +1,78 @@
+import React, { useState } from "react";
+import { Container, Row, Col } from "react-bootstrap";
+import "./index.scss";
+import { useTranslation } from "react-i18next";
+import OpportunitiesCard from "../OpportunitiesCard";
+import SearchBar from "../SearchBar";
+
+const JobsTrainingContent = () => {
+  const { t } = useTranslation();
+  const [translationKey, setTranslationKey] = useState("jobOpportunities");
+  const cardsArr = [...t(`home.${translationKey}`, { returnObjects: true })];
+  const cards = cardsArr.map((opportunity) => (
+    <OpportunitiesCard {...opportunity} key={opportunity.id} />
+  ));
+
+  const cardsList = cards.map((card, index) => {
+    return (
+      <Col
+        sm={12}
+        md={6}
+        className={index % 2 === 0 ? "peachBorder" : "greenBorder"}
+        key={card.props.id}
+      >
+        {card}
+      </Col>
+    );
+  });
+
+  const [isDisabled, setIsDisabled] = useState(true);
+
+  const selectButton = () => {
+    setIsDisabled(!isDisabled);
+  };
+
+  return (
+    <Container className="jobsTrainingContent">
+      <Row className="searchBar">
+        <SearchBar />
+      </Row>
+      <Row>
+        <Col md={3}>
+          <div>
+            <button
+              disabled={isDisabled}
+              type="button"
+              translationKey="button"
+              className="jobsBtn"
+              onClick={() => {
+                selectButton();
+                setTranslationKey("jobOpportunities");
+              }}
+            >
+              Job Opportunities
+            </button>
+          </div>
+          <div>
+            <button
```

Comment:
this is another instance of copy-pasted code here if i told you, for example, you had to delete line 59 (which i think you do), you have to remember to do it in 2 places

a thing that happens a lot in reviewing this type of code:

me: delete line 59
them: ok
them: forgets to do it on line 46 (bc the code was copy-pasted. also look at the most recent comments on jamil's PR for an example of this happening)

but also, sometimes "me" is not louis but yourself, when you're making changes to your own code

basically, move the button into either a function `const opportunityButton(buttonText, translationKey)` or make it another component (slightly cleaner) (it can be in the same file, some people want only one comp. per file but i don't care esp. if the component is not used outside the file)

Reply:
Actually, I took the code from the JobsTrainingHome so it used somewhere else. I will create another component so we can add to home page too 👍

Reply:
@louisrli I did a separate PR for the button so this PR won't become so big. After merging I can push this code again